README.md

```diff
@@ -22,17 +22,32 @@ The following template parameters are set as environment variables in the lambda
 
 ## Triggering
 
-The CloudFormation template will output the endpoint URL that can be loaded in a browser, e.g.:
-`https://57lejqxw32.execute-api.us-east-1.amazonaws.com/Prod/all/costcenters.json`
+The CloudFormation template will output endpoint URLs for each supported API route.
 
-## Respones Format
+### Retrieving Cost Centers
+
+Currently only one output format is supported: an unfiltered mapping of all cost center codes.
+The endpoint URL for this format is given in the output `ApiRouteAllCostCenters`, e.g.:
+```
+https://57lejqxw32.execute-api.us-east-1.amazonaws.com/Prod/all/costcenters.json
+```
+#### Respones Format
 
 The API will return a json string representing a dictionary mapping program codes to their names.
 E.g.:
 ```json
 {"000000": "No Program", "990300": "Program Infrastructure"}
 ```
 
+### Cache Administration
```

Comment:
can you please create a section to explain the purpose for creating a cache for this simple service?  

Also are you planning to integrate the nice new caching images into the docs somewhere?